examples/resources/hpegl_metal_project/README.md

```diff
@@ -1,5 +1,5 @@
 # Example of creating project
-**IMPORTANT NOTE**: Projects can be created only with an IAM token and a BMaaS Owner role associated with the token
+**IMPORTANT NOTE**: Projects can be created only with an IAM token and a Metal Owner role associated with the token
```

Comment:
I don't think this is correct. We still use BMAAS in the IAM roles.

Reply:
yes, Neil is correct. The roles on the GLC side have 'BMaaS', so leave this line as it is.

Reply:
Going to mark this changes requested for the IAM Role